root/static/scripts/edit/URLCleanup.js

```diff
@@ -2025,7 +2025,7 @@ const CLEANUPS = {
       new RegExp("^(https?://)?(www\\.)?flattr\\.com/profile/[^/?#]", "i"),
       new RegExp("^(https?://)?(www\\.)?patreon\\.com/[^/?#]", "i"),
       new RegExp("^(https?://)?(www\\.)?paypal\\.me/[^/?#]", "i"),
-      new RegExp("^(https?://)?(www\\.)?tipeee\\.com/[^/?#]", "i"),
+      new RegExp("^(https?://)?(www\.)?(?:(?:fr|en|de|es|it)\.)?tipeee\.com/[^/?#]", "i"),
```

Comment:
It is probably not needed to match `www.en.tipeee` but it's still better than the previous version of this RegExp.

Reply:
Okay, would it be better as:
[...] https?://?`((?:www|fr|en|de|es|it)\.)`?tipeee\.com [...]

Reply:
Yes, but if you want it to keep working for any additional language in the future, just use `(?:[^/]+\\.)?tipeee\\.com` instead, that will match any subdomain.

Note that dot should be escaped twice `\\.` here.

Reply:
That sounds best. I'll commit that this evening :)